lib/Service/FeedServiceV2.php

```diff
@@ -246,6 +246,10 @@ public function create(
             $feed->setTitle($title);
         }
 
+        if (!$feed->getTitle()) {
```

Comment:
This is probably a clearer indication of what we want.
```suggestion
        } else {
```